File: Reminders/Core Data/Models/RIReminder+CoreDataProperties.m
//
//  RIReminder+CoreDataProperties.m
//  Reminders
//
//  Created by Ostap Tyvonovych on 2/26/20.
//  Copyright © 2020 Ostap Tyvonovych. All rights reserved.
//
//

#import "RIReminder+CoreDataProperties.h"

@implementation RIReminder (CoreDataProperties)

+ (NSFetchRequest<RIReminder *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:NSStringFromClass(RIReminder.class)];
}

@dynamic arrayOfImagePaths;
@dynamic date;
@dynamic text;

@end

File: Reminders/RIReminder+CoreDataProperties.swift
//
//  RIReminder+CoreDataProperties.swift
//  Reminders
//
//  Created by Ostap Tyvonovych on 2/26/20.
//  Copyright © 2020 Ostap Tyvonovych. All rights reserved.
//
//

import Foundation
import CoreData


extension RIReminder {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<RIReminder> {
        return NSFetchRequest<RIReminder>(entityName: "RIReminder")
    }


}
